.env.example

```diff
@@ -5,8 +5,8 @@ NEXT_PUBLIC_CLERK_SIGN_UP_URL=/join
 NEXT_PUBLIC_CLERK_AFTER_SIGN_IN_URL=/dashboard
 NEXT_PUBLIC_CLERK_AFTER_SIGN_UP_URL=/welcome
 
-NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=
-CLERK_SECRET_KEY=
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_test_cmVuZXdpbmctdGV0cmEtOTguY2xlcmsuYWNjb3VudHMuZGV2JA
```

Comment:
Dont expose these keys, Please add this file in .gitIgnore

Reply:
I have updated the .env.example file.